freud/cluster.pyx

```diff
@@ -183,6 +183,18 @@ cdef class ClusterProperties(_Compute):
         this method, these properties can be accessed with the
         :code:`centers` and :code:`gyrations` attributes.
 
+        Example::
+
+            >>> # Assuming box, positions, and nlist have already been defined
+            >>> # Compute clusters using box, positions, and nlist data
+            >>> import freud
+            >>> import numpy as np
+            >>> cl = freud.cluster.Cluster()
+            >>> cl.compute(box, positions, neighbors=nlist)
```

Comment:
You may want to include a comment here that explains (briefly) why you need to calculate this first.
```suggestion
            >>> # Determine clusters
            >>> cl = freud.cluster.Cluster()
            >>> cl.compute(box, positions, neighbors=nlist)
```